src/imitation/scripts/download_experts.py

```diff
@@ -0,0 +1,32 @@
+import urllib.request
+import zipfile
+
+experts_url = ("https://www.dl.dropboxusercontent.com/"
```

Comment:
FYI this zip contains a bunch of https://en.wikipedia.org/wiki/.DS_Store files, you may want to add that to the exclude list when generating the ZIP

Reply:
Thanks for catching this, I'll keep the exclude in mind next time I generate